src/toil/leader.py

```diff
@@ -464,8 +464,18 @@ def getJob(jobId):
                     self.successorJobStoreIDToPredecessorJobs[successorJobStoreID].append(jobWrapper)
 
 class FailedJobsException( Exception ):
-    def __init__(self, jobStoreLocator, numberOfFailedJobs):
-        super( FailedJobsException, self ).__init__( "The job store '%s' contains %i failed jobs" % (jobStoreLocator, numberOfFailedJobs))
+    def __init__(self, jobStoreLocator, numberOfFailedJobs, failedJobs, jobStore):
```

Comment:
This signature makes no sense. If you are passing failedJobs, you don't need to pass its length in a separate argument.